cms/admin/placeholderadmin.py

```diff
@@ -579,24 +594,43 @@ def move_plugin(self, request):
                         return HttpResponseBadRequest(force_text(
                             _('parent must be in the same language as '
                               'plugin_language')))
-                    plugin.parent_id = parent.pk
-                    plugin.language = language
-                    plugin.save()
+                    plugin = plugin.update(refresh=True, parent=parent, **plugin_data)
```

Comment:
afaik doesn't make much difference, unless necessary I prefer to not use the more internal `_id` approach.